main/tray.js

```diff
@@ -29,6 +47,13 @@ function init () {
   windows.main.on('hide', updateTrayMenu)
 }
 
+function checkLinuxTraySupport (cb) {
+  cp.exec('dpkg --get-selections libappindicator1', function (err, stdout) {
```

Comment:
Yeah I know --- if `dpkg` isn't present, then `err` will be set in the callback & it'll assume that tray support isn't there.

Adding `libappindicator1` as a dependency in the deb file sounds good to me. I added this check for the people who install from the zip file.